### ISSUE TITLE
text wrap is broken by default

### ISSUE THREAD

**Issue by rahul-gj** [cmderdev/cmder#2296]
I recently installed cmder as I was tired by windows cmd. The main issue with cmd was 'utf-8' and text wrapping. It doesn't allow it.

I am using cmder since last 3 days. I think there might be the problem with cmder text wrapping functionality.

demo:
before:
![image](https://user-images.githubusercontent.com/8735524/78879811-0d867f00-7a72-11ea-89d0-e6479a63c6b9.png)

intermediate:
![image](https://user-images.githubusercontent.com/8735524/78879919-37d83c80-7a72-11ea-9eda-5685d334aa95.png)

after:
![image](https://user-images.githubusercontent.com/8735524/78880049-681fdb00-7a72-11ea-9892-c08aa1a40e08.png)

In short, while resizing the window, the text wrapping doesn't work. If that doesn't work that is still OK but here even the text is lost.

**Comment by daxgames** [comment]
Works for me.

**Comment by daxgames** [comment]
Need more info to reproduce.  Run `cmder_diag` and post output.  

My guess though is it may be a Conemu thing since Conemu is the terminal emulator Cmder uses.  If you could try cownloading and running Conemu outside Cmder and see if it happens there also that would help narrow things down.

**Comment by rahul-gj** [comment]
```
λ cmder_diag
------------------------------------
dir "C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\dist\cmder_mini"
------------------------------------
 Volume in drive C has no label.
 Volume Serial Number is AE4D-A0D2

 Directory of C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\dist\cmder_mini

09/04/20  15:13    <DIR>          .
09/04/20  15:13    <DIR>          ..
09/04/20  15:13    <DIR>          bin
09/04/20  15:13           141,824 Cmder.exe
09/04/20  15:58    <DIR>          config
09/04/20  15:13    <DIR>          icons
09/04/20  15:13             1,069 LICENSE
09/04/20  15:13    <DIR>          vendor
09/04/20  15:13                 0 Version 1.3.14.982
               3 File(s)        142,893 bytes
               6 Dir(s)  122,304,716,800 bytes free

------------------------------------
dir "C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\dist\cmder_mini\vendor"
------------------------------------
 Volume in drive C has no label.
 Volume Serial Number is AE4D-A0D2

 Directory of C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\dist\cmder_mini\vendor

09/04/20  15:13    <DIR>          .
09/04/20  15:13    <DIR>          ..
09/04/20  15:13    <DIR>          bin
09/04/20  15:13    <DIR>          clink
09/04/20  15:13    <DIR>          clink-completions
09/04/20  15:13            13,686 clink.lua
09/04/20  15:13             4,015 clink_settings.default
09/04/20  15:13             3,903 cmder_exinit
13/04/20  09:58    <DIR>          conemu-maximus5
09/04/20  15:13            54,990 ConEmu.xml.default
09/04/20  15:13            15,119 init.bat
09/04/20  15:13    <DIR>          lib
09/04/20  15:13             7,153 profile.ps1
09/04/20  15:13    <DIR>          psmodules
09/04/20  15:13                49 Readme.md
09/04/20  15:13               720 sources.json
09/04/20  15:13               443 user_aliases.cmd.default
09/04/20  15:13               722 user_profile.cmd.default
09/04/20  15:13               861 user_profile.ps1.default
09/04/20  15:13               295 user_profile.sh.default
              12 File(s)        101,956 bytes
               8 Dir(s)  122,304,716,800 bytes free

------------------------------------
dir /s "C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\dist\cmder_mini\bin"
------------------------------------
 Volume in drive C has no label.
 Volume Serial Number is AE4D-A0D2

 Directory of C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\dist\cmder_mini\bin

09/04/20  15:13    <DIR>          .
09/04/20  15:13    <DIR>          ..
09/04/20  15:13                84 Readme.md
               1 File(s)             84 bytes

     Total Files Listed:
               1 File(s)             84 bytes
               2 Dir(s)  122,304,716,800 bytes free

------------------------------------
dir /s "C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\dist\cmder_mini\config"
------------------------------------
 Volume in drive C has no label.
 Volume Serial Number is AE4D-A0D2

 Directory of C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\dist\cmder_mini\config

09/04/20  15:58    <DIR>          .
09/04/20  15:58    <DIR>          ..
13/04/20  13:03                18 .history
09/04/20  15:13    <DIR>          profile.d
09/04/20  15:13               887 Readme.md
09/04/20  15:13             4,015 settings
09/04/20  15:58            51,298 user-ConEmu.xml
09/04/20  15:13               443 user_aliases.cmd
09/04/20  15:13               722 user_profile.cmd
               6 File(s)         57,383 bytes

 Directory of C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\dist\cmder_mini\config\profile.d

09/04/20  15:13    <DIR>          .
09/04/20  15:13    <DIR>          ..
               0 File(s)              0 bytes

     Total Files Listed:
               6 File(s)         57,383 bytes
               5 Dir(s)  122,304,716,800 bytes free

------------------------------------
set
------------------------------------
add_path=C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\dist\cmder_mini
aliases=C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\dist\cmder_mini\config\user_aliases.cmd
ALLUSERSPROFILE=C:\ProgramData
ANSICON=168x1000 (168x42)
ANSICON_DEF=7
APPDATA=C:\Users\rahulkumar.patel\AppData\Roaming
architecture=64
architecture_bits=64
ccall=call C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\dist\cmder_mini\vendor\bin\cexec.cmd
cexec=C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\dist\cmder_mini\vendor\bin\cexec.cmd
ChocolateyInstall=C:\ProgramData\chocolatey
ChocolateyLastPathUpdate=Mon Aug  6 10:04:01 2018
ChocolateyToolsLocation=C:\tools
CMDER_ALIASES=1
CMDER_CLINK=1
CMDER_CONFIGURED=1
CMDER_INIT_END=13:03:37.26
CMDER_INIT_START=13:03:21.63
CMDER_ROOT=C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\dist\cmder_mini
CMDER_SHELL=cmd
CMDER_USER_FLAGS=
CommonProgramFiles=C:\Program Files\Common Files
CommonProgramFiles(x86)=C:\Program Files (x86)\Common Files
CommonProgramW6432=C:\Program Files\Common Files
COMPUTERNAME=C014247
ComSpec=C:\Windows\system32\cmd.exe
ConEmuANSI=ON
ConEmuAnsiLog=
ConEmuArgs=
ConEmuArgs2=
ConEmuBackHWND=0x001D0634
ConEmuBaseDir=C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\dist\cmder_mini\vendor\conemu-maximus5\ConEmu
ConEmuBaseDirShort=C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\dist\cmder_mini\vendor\conemu-maximus5\ConEmu
ConEmuBuild=191012
ConEmuCfgDir=C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\dist\cmder_mini\vendor\conemu-maximus5
ConEmuConfig=
ConEmuDir=C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\dist\cmder_mini\vendor\conemu-maximus5
ConEmuDrawHWND=0x001805C2
ConEmuDrive=C:
ConEmuHooks=Enabled
ConEmuHWND=0x000505D8
ConEmuPalette=Monokai
ConEmuPID=6108
ConEmuServerPID=6412
ConEmuTask={cmder::cmd}
ConEmuWorkDir=C:\Users\rahulkumar.patel
ConEmuWorkDrive=C:
debug_output=0
depth=1
fast_init=0
feFlagName=
find_query=C:\\Users\\rahulkumar.patel\\AppData\\Roaming\\plrplus\\dist\\cmder_mini
found=1
FP_NO_HOST_CHECK=NO
git_executable=C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\bin\PortableGit\bin\git.exe
GIT_INSTALL_ROOT=C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\bin\PortableGit
git_locale="C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\bin\PortableGit\usr\bin\locale.exe"
GIT_VERSION_USER=2.22.0.windows.1
HOME=C:\Users\rahulkumar.patel
HOMEDRIVE=C:
HOMEPATH=\Users\rahulkumar.patel
K2PDFOPT_CUSTOM0=Last Settings;-dev nex7 -fc- -as;
K2PDFOPT_CUSTOM1=2-column paper;-mode 2col;
K2PDFOPT_CUSTOM2=Trim Margins;-mode fw;
K2PDFOPT_WINPOS=470 0 1362 738
LANG=en_US.UTF-8
lib_base=call "C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\dist\cmder_mini\vendor\lib\lib_base.cmd"
lib_console=call "C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\dist\cmder_mini\vendor\lib\lib_console.cmd"
lib_git=call "C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\dist\cmder_mini\vendor\lib\lib_git.cmd"
lib_path=call "C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\dist\cmder_mini\vendor\lib\lib_path.cmd"
lib_profile=call "C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\dist\cmder_mini\vendor\lib\lib_profile.cmd"
LOCALAPPDATA=C:\Users\rahulkumar.patel\AppData\Local
LOGONSERVER=\\BAD-AD
max_depth=1
mylib=C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\py\myModule
nix_tools=1
NUMBER_OF_PROCESSORS=4
OLD_PATH=C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\dist\cmder_mini\bin;C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\dist\cmder_mini\vendor\bin;C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\dist\cmder_mini\vendor\conemu-maximus5\ConEmu\Scripts;C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\dist\cmder_mini\vendor\conemu-maximus5;C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\dist\cmder_mini\vendor\conemu-maximus5\ConEmu;C:\Program Files (x86)\PerkinElmerInformatics\ChemOffice2016\ChemScript\Lib;C:\Program Files (x86)\Common Files\Oracle\Java\javapath;C:\ProgramData\Oracle\Java\javapath;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Program Files\Microsoft SQL Server\130\Tools\Binn\;C:\PostgreSQL\pg96\bin;C:\PostgreSQL\pg10\bin;C:\Users\rahulkumar.patel\AppData\Local\Programs\Python\Python38\Scripts\;C:\Users\rahulkumar.patel\AppData\Local\Programs\Python\Python38\;D:\PLR\bin;C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\bin\PortableGit\bin;C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\dist\pgsql\bin;C:\Program Files (x86)\Google\Chrome\Application;C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\bin\PortableGit\cmd;C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\bin\PortableGit\mingw64\bin;C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\bin\PortableGit\usr\bin;C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\dist\cmder_mini
OneDrive=C:\Users\rahulkumar.patel\OneDrive - Zydus
OneDriveCommercial=C:\Users\rahulkumar.patel\OneDrive - Zydus
OS=Windows_NT
Path=C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\dist\cmder_mini\bin;C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\dist\cmder_mini\vendor\bin;C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\dist\cmder_mini\vendor\conemu-maximus5\ConEmu\Scripts;C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\dist\cmder_mini\vendor\conemu-maximus5;C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\dist\cmder_mini\vendor\conemu-maximus5\ConEmu;C:\Program Files (x86)\PerkinElmerInformatics\ChemOffice2016\ChemScript\Lib;C:\Program Files (x86)\Common Files\Oracle\Java\javapath;C:\ProgramData\Oracle\Java\javapath;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Program Files\Microsoft SQL Server\130\Tools\Binn\;C:\PostgreSQL\pg96\bin;C:\PostgreSQL\pg10\bin;C:\Users\rahulkumar.patel\AppData\Local\Programs\Python\Python38\Scripts\;C:\Users\rahulkumar.patel\AppData\Local\Programs\Python\Python38\;D:\PLR\bin;C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\bin\PortableGit\bin;C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\dist\pgsql\bin;C:\Program Files (x86)\Google\Chrome\Application;C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\bin\PortableGit\cmd;C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\bin\PortableGit\mingw64\bin;C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\bin\PortableGit\usr\bin;C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\dist\cmder_mini
PATHEXT=.COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC
path_position=append
PLINK_PROTOCOL=ssh
PROCESSOR_ARCHITECTURE=AMD64
PROCESSOR_IDENTIFIER=Intel64 Family 6 Model 42 Stepping 7, GenuineIntel
PROCESSOR_LEVEL=6
PROCESSOR_REVISION=2a07
ProgramData=C:\ProgramData
ProgramFiles=C:\Program Files
ProgramFiles(x86)=C:\Program Files (x86)
ProgramW6432=C:\Program Files
PROMPT=$p$_$+$g
PSModulePath=;C:\Users\rahulkumar.patel\AppData\Local\Google\Cloud SDK\google-cloud-sdk\platform\PowerShell
PUBLIC=C:\Users\Public
PYTHONPATH=C:\Program Files (x86)\PerkinElmerInformatics\ChemOffice2016\ChemScript\Lib
SESSIONNAME=Console
SVN_SSH=C:\\Users\\rahulkumar.patel\\AppData\\Roaming\\plrplus\\bin\\PortableGit\\bin\\ssh.exe
SystemDrive=C:
SystemRoot=C:\Windows
TEMP=C:\Users\RAHULK~1.PAT\AppData\Local\Temp
TERM=cygwin
time_init=0
TMP=C:\Users\RAHULK~1.PAT\AppData\Local\Temp
USERDNSDOMAIN=ZYDUSCADILA.COM
USERDOMAIN=ZYDUSCADILA
USERNAME=Rahulkumar.Patel
USERPROFILE=C:\Users\rahulkumar.patel
user_aliases=C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\dist\cmder_mini\config\user_aliases.cmd
USER_BUILD=windows.1
USER_MAJOR=2
USER_MINOR=22
USER_PATCH=0
verbose_output=0
VS140COMNTOOLS=C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\Tools\
windir=C:\Windows
windows_tracing_flags=3
windows_tracing_logfile=C:\BVTBin\Tests\installpackage\csilogfile.log

------------------------------------
where git
------------------------------------
C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\bin\PortableGit\bin\git.exe
C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\bin\PortableGit\cmd\git.exe
C:\Users\rahulkumar.patel\AppData\Roaming\plrplus\bin\PortableGit\mingw64\bin\git.exe

------------------------------------
Make sure you sanitize this output of private data prior to posting it online for review by the CMDER Team!
------------------------------------
```

**Comment by rahul-gj** [comment]
The problem is, The text is not free flowing. It stays in the same line no matter you resize it or not.

Take this screen shots as an example

Original
![image](https://user-images.githubusercontent.com/8735524/79102197-df61b180-7d87-11ea-9712-40e9ac0038d6.png)

After decreasing size:
![image](https://user-images.githubusercontent.com/8735524/79102301-16d05e00-7d88-11ea-9834-82ef1e23d23e.png)


After maximizing again.
![image](https://user-images.githubusercontent.com/8735524/79102378-43847580-7d88-11ea-94f7-7613ed9fefd9.png)


It is not useful at all.

**Comment by daxgames** [comment]
@rahul-gj It works properly for me.  I am not seeing what you are seeing.  When I resize smaller the text moves and wraps.  When I resize larger the text moves and unwraps.

**Comment by rahul-gj** [comment]
I am on windows 7. I am using default settings. I have already sent you the screenshots. If anyone with the windows 7 able to solve the issue, please suggest. I have already attached screenshots to prove that this is really happening and I am not making it up. If any further clarification required please suggest.

**Comment by daxgames** [comment]
@rahul-gj I never suggested you made it up, I just can't reproduce it.  If I can't reproduce it, I can't even begin to try and fix it.

The only suggestions I have for you is to try it with a version of Conemu without Cmder running straight cmd.exe.  Conemu is the terminal emulator/GUI for Cmder and is responsible for displaying text.  We, the Cmder devs, do not maintain Conemu.

Check the Conemu issues both open and closed to see if the problem is reported there.

**Comment by rahul-gj** [comment]
I have tried Latest version of Conemu. It is having same effect. So it is indeed conemu problem. Thanks.
